org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/tests/test.js

```diff
@@ -0,0 +1,107 @@
+var test = require('ava');
+var q = require('q');
+var OpenT2T = require('opent2t').OpenT2T;
+var config = require('./testConfig');
+
+console.log('Device Under Test -  Name: ' + config.Device.name + '  Props: ' + JSON.stringify(config.Device.props));
+var translatorPath = require('path').join(__dirname, '..');
+
+///
+/// Run a series of tests to validate the translator
+///
+
+// Get AmbientTemperature
+test.serial('AmbientTemperature', t => {

```

Comment:
@jasongin turns out the tests are identical for both nest and wink thermostats :) since they both implement the same schema. I spoke to @arjun-msft and we agreed to at least file an issue to CONSIDER making the tests common... would be nice to ensure via these tests that all the translators for a particular schema work the same way. I will assign the issue to @arjun-msft and he can come up with a design after thinking through pros/cons.